File: tests/python/createComment.py
import requests
import json
import sys

jsonBody = {
	"thread_id": 1,
	"author": "admin",
	"comment": "some comment"
}

headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
res = requests.post('http://localhost:3000/createComment', data = json.dumps(jsonBody), headers = headers)
print(res.json())

File: tests/nodejs/comment/createComment.js
var request = require('request');

var jsonBody = {
	thread_id: 1,
	author: "admin",
	comment: "some comment"
}

request({
	url: 'http://localhost:3000/createComment',
	method: 'post',
	json: true,
	body: jsonBody
}, function (error, response, body) {
	console.log(response);
});